pytun_pmd3/wintun.py

```diff
@@ -154,6 +161,16 @@ class MIB_UNICASTIPADDRESS_ROW(Structure):
 iphlpapi.CreateUnicastIpAddressEntry.argtypes = [POINTER(MIB_UNICASTIPADDRESS_ROW)]
 iphlpapi.CreateUnicastIpAddressEntry.restype = DWORD
 
+kernel32.WaitForSingleObject.restype = DWORD
+kernel32.WaitForSingleObject.argtypes = [HANDLE, DWORD]
+
+
+def run_in_executor(func, *args, **kwargs):
```

Comment:
Just use `asyncio.to_thread` instead of this function
https://docs.python.org/3/library/asyncio-task.html#asyncio.to_thread

Reply:
didn't know this function. done :)